sdks/java/harness/src/test/java/org/apache/beam/fn/harness/state/FakeBeamFnStateClient.java

```diff
@@ -174,6 +215,124 @@ public CompletableFuture<StateResponse> handle(StateRequest.Builder requestBuild
         response = StateResponse.newBuilder().setAppend(StateAppendResponse.getDefaultInstance());
         break;
 
+      case ORDERED_LIST_GET:
+        {
+          long start = request.getOrderedListGet().getRange().getStart();
+          long end = request.getOrderedListGet().getRange().getEnd();
+
+          KvCoder<Long, Integer> coder = KvCoder.of(VarLongCoder.of(), VarIntCoder.of());
+          long sortKey = start;
+          int index = 0;
+          if (request.getOrderedListGet().getContinuationToken().size() > 0) {
```

Comment:
Use isEmpty?

Reply:
The code here is to check if there is a non-empty continuation token. An empty continuation token in the get request suggests it should return the first block in the query range. Non-empty continuation token here contains information of which block it is requesting.

Reply:
I'm just suggesting `String::isEmpty()` over `String::size() > 0`.

Reply:
Ack!